tests/api/ask_astro/rest/controllers/test_get_requests.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import AsyncMock, Mock, patch
+from uuid import uuid4
+
+import pytest
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "mock_exists,mock_data,expected_status,expected_response",
```

Comment:
nitpick: add space for easier reading
```suggestion
    "mock_exists, mock_data,expected_status, expected_response",
```

Reply:
https://github.com/astronomer/ask-astro/pull/77/commits/455e1a42e21005be7bb3d0cf066cf2ff924f97b3